### ISSUE TITLE
[LibreTranslator/Kooperationsvereinbarung] 5. Zusammenarbeit

### ISSUE THREAD

**Issue by kindustrii** [deeptensornn/DeepSimpleL#12]
Siehe Issue #6 

Analog DeepSimpleL, werden wir auch für diese Ideeneinreichung (LibreTranslator) einen zusätzlichen Partner benötigen. Wäre das für den Jedermann-Verlag auch von Interesse? Oder gibt eine andere Idee?

**Comment by deeptensornn** [comment]
Ja, da will ich am Montag sehen ob ich das klären kann. Hab ja an das BMAS geschrieben wegen Mitarbeit eines Verlages. Man kann ja KMUs ankreuzen. Wie du weißt, steht das für 'kleines oder mittleres Unternehmen'. Nun Jedermann eher als M bewegt sich ausschließlich im deutschen Sprachraum. Ich muss mal sehen ...

Noch Klärungsbedarf
